vllm/model_executor/models/idefics3.py

```diff
@@ -610,7 +606,33 @@ def forward(
 @MULTIMODAL_REGISTRY.register_max_image_tokens(get_max_idefics3_image_tokens)
 @INPUT_REGISTRY.register_dummy_data(dummy_data_for_idefics3)
 @INPUT_REGISTRY.register_input_processor(input_processor_for_idefics3)
-class Idefics3ForConditionalGeneration(nn.Module, SupportsMultiModal):
+class Idefics3ForConditionalGeneration(nn.Module, SupportsMultiModal,
```

Comment:
We also need to update the LoRA support documentation

Reply:
You're right, I'll update the documentation.